README.md

```diff
@@ -42,24 +42,25 @@ Select *Git* then configure:
 
 Select *Stash Pull Request Builder* then configure:
 
-- **Cron**: must be specified. eg: every 2 minute `H/2 * * * *`
-- **Stash URL**: the *http* or *https* URL of the Stash REST API Host (NOT *ssh*). eg: *https://example.com*
-- **Stash Credentials**: Select or Add the login username/password for the Stash REST API Host (NOT ssh key)
-- **Project**: abbreviated project code. eg: *PRJ* or *~user*
-- **RepositoryName**: eg: *Repo*
+- **Cron schedule**: How often to poll, e.g. every 2 minute: `H/2 * * * *`
+- **Stash URL**: The *http* or *https* URL of the Stash REST API (*NOT* ssh), e.g. *https://stash.example.com/*
```

Comment:
General question for numerous places in the texts: shouldn't HTTP, HTTPS, SSH be upper-cased (they are abbreviations after all)? Especially since SSL was upper-cased by this PR ;)

Reply:
I believe it was possible to select an ssh key for the Stash account, which was confusing. That required extra clarifications. The issue with the credential selection was fixed in version 1.7.0, but the clarifications are still there. We could tone them down slightly.

I'd rather remove reverences to http and https completely. I'm also thinking about removing all references to REST. No need to add an abbreviation that doesn't help understanding the text. Stash API is an API to Stash, that's it.

Standards use `HTTP`, but `http` looks better to me, as I see the lowercase version in URLs all the time.

I'm leaving that part as is for now.

Reply:
> I'd rather remove reverences to http and https completely. I'm also thinking about removing all references to REST. No need to add an abbreviation that doesn't help understanding the text. Stash API is an API to Stash, that's it.

I'm not sure, I still find it useful to have a clarification that it's a HTTP URL and not and SSH Clone URL. On the other hand you can still use HTTP Clone URL. 

> Standards use HTTP, but http looks better to me, as I see the lowercase version in URLs all the time.

Well, HTTP is the abbreviated name of the protocol. `http` is an URI/URL scheme value

Reply:
Let's get back to this point later as not to block the other changes